### PR TITLE
Add ssl_ca_cert option to backend

### PR DESCRIPTION
**Pull request by tomyan**
Make it possible to configure the CA cert used to verify the backend
for a Fastly service.

### CODE REVIEW

File: fastly/resource_fastly_service_v1.go [hunk 1]
```diff
@@ -271,6 +271,12 @@ func resourceServiceV1() *schema.Resource {
 							Description: "SSL certificate hostname",
 							Deprecated:  "Use ssl_cert_hostname and ssl_sni_hostname instead.",
 						},
+						"ssl_ca_cert": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "",
```
Comment (catsby) [inline fastly/resource_fastly_service_v1.go:277]: Is the empty string default here needed?

**Review by catsby — COMMENTED** [review]
Question about the empty default, does it serve a purpose or was a test failing without it?

File: fastly/resource_fastly_service_v1.go (hunk 1 above)
Comment (tomyan) [reply fastly/resource_fastly_service_v1.go:277]: The tests pass without the default value. I put it in to match the style of similar optional parameters - e.g. `ssl_cert_hostname`.

Thanks

Tom

**Comment by tomyan** [comment]
@catsby just bumping the main thread in case you didn't see my response to your comment in the code. Let me know if it looks okay, or whether any changes are needed. Thanks

**Review by catsby — APPROVED** [review]
Thanks!

**Pull request merged**